### PR TITLE
Make some minor tweaks to dashboard based on Eryn's feedback

### PR DESCRIPTION
**Pull request by meowwwls**
Just a few small changes based on Eryn's latest feedback for the dashboard. Some might require reverting or discussion, but I wanted to tackle a few easy fixes:

- Changed the icon on the Edit Donation card to use Eryn's (kept the hover style because changing it to `puddle` was barely any difference from white, making the change almost imperceivable; can circle back to this)
- Changed the text sizing and weight on the dashboard cards
- Changed the look of the 'Download Data' button, but kept the icon
- Changed some copy on the dashboard to reflect Eryn's requests
- Removed the underline from the banner links
- Removed the border from the close icon on the banner (I personally don't agree with this move or the one above, but will leave them for now or can change them back)
- Commented out the extra links that were added to the auth nav so if we decide to keep them, it's easy to undo. I left the icons because I think it looks better, until we can discuss. 
- Adjusted the spacing on footer links and changed them the be the same as the nav links
- Centered the text below the login / sign up forms
- Tighten the line height on subheadings
- Increase letter spacing in uppercase headings

### CODE REVIEW

File: components/common/footer.js [hunk 1]
```diff
@@ -12,12 +12,12 @@ const loggedOutLinks = [
     text: 'About Us'
   },
   {
-    url: '/how-it-works',
-    text: 'How It Works'
+    url: '/faq',
+    text: 'FAQs'
   },
   {
-    url: '/contact',
```
Comment (joelwass) [inline components/common/footer.js:19]: are we removing contact?

File: components/common/header/authNav.js [hunk 2]
```diff
@@ -82,7 +82,8 @@ const AuthNav = ({ user, onLogout }) => {
             />
           </MenuButton>
           <MenuList backgroundColor='lightRock'>
-            <MenuItem
+            {/* TODO: commenting these out for now in case we decide to add them back, but I agree with Eryn that they aren't needed */}
```
Comment (joelwass) [inline components/common/header/authNav.js]: i'd prefer we have these, especially as our product is young

File: components/dashboard/donationCard.js [hunk 3]
```diff
@@ -58,15 +58,17 @@ const DonationCard = ({ donationAmount, donationLoading, hasDonation, optOutOfAd
...
+              icon='pencil'
               title='Edit monthly donation'
             />
           </Flex>
         )}
         <UnderlinedHeading
           as='h3'
           id='donation-amt'
-          text='Monthly Donation'
+          text='Donated monthly'
```
Comment (joelwass) [inline components/dashboard/donationCard.js]: i dont think donated monthly implies what the card shows. the card shows an active donation, nothing past tense about it, let's revert the text change

File: components/dashboard/topTenPackagesView.js [hunk 4]
```diff
@@ -17,9 +17,7 @@ const TopTenPlaceholder = () => (
   <>
     <Icon name='bars' size='15rem' marginBottom='3rem' />
     <Text>
-      Once you've installed <strong>Flossbank</strong> and start installing Open
-      Source packages, you'll be able to see data about the top ten packages you
-      use.
+      Once you've installed the package manager, you'll be able to see data about your top packages.
```
Comment (joelwass) [inline components/dashboard/topTenPackagesView.js]: revert since this isn't exactly accurate. you need to install the pmw AND install open source packages

File: components/dashboard/topTenPackagesView.js [hunk 5]
```diff
@@ -60,7 +59,7 @@ const TopTenPackagesView = ({ topTenPackages }) => (
         display='flex'
         alignItems='center'
       >
-        Top Ten Packages Used
+        Top Packages
```
Comment (joelwass) [inline components/dashboard/topTenPackagesView.js]: revert - also don't think it conveys the info accurately

File: pages/dashboard.js [hunk 6]
```diff
@@ -152,7 +152,6 @@ const Dashboard = () => {
             Looks like you haven't installed the package manager wrapper yet.
             Head over to{' '}
             <TextLink
-              textDecoration='underline'
```
Comment (joelwass) [inline pages/dashboard.js]: i personally find it hard to tell it's a link without the underline since the text is the same color as the surrounding text. what do you think?

File: components/common/banner.js [hunk 7]
```diff
@@ -29,7 +29,6 @@ const Banner = ({ icon, children, onCloseClick }) => {
       {children}
       <CloseButton
         onClick={onCloseClick}
-        border='2px solid'
```
Comment (joelwass) [inline components/common/banner.js:32]: i agree with you, i liked the border

File: components/splash/cards/whatIsFBCards.js [hunk 8]
```diff
@@ -37,16 +37,17 @@ const Card = ({ icon, text, heading }) => (
     fontSize='0.875rem'
     lineHeight='1.3'
   >
+    <Icon name={icon} size='1.25rem' margin='0 0 .75rem' />
```
Comment (joelwass) [inline components/splash/cards/whatIsFBCards.js]: why this change? i personally don't like it as much, what do you think?

**Review by joelwass — CHANGES_REQUESTED** [review]

File: pages/dashboard.js (hunk 6 above)
Comment (joelwass) [reply pages/dashboard.js]: pete has a change to this that helps more by making the outter text "rock" color

File: components/common/header/authNav.js (hunk 2 above)
Comment (joelwass) [reply components/common/header/authNav.js]: also now there's no way to get back to the dashboard from the splash page (if a user gets there)

File: components/dashboard/donationCard.js (hunk 3 above)
Comment (joelwass) [reply components/dashboard/donationCard.js]: pete and i worked on wording in his PR - keep those changes

File: components/common/banner.js (hunk 7 above)
Comment (stripedpajamas) [reply components/common/banner.js:32]: i like it without the border :D

File: components/common/footer.js [hunk 9]
```diff
@@ -12,12 +12,12 @@ const loggedOutLinks = [
...
-    text: 'Contact Us'
+    url: '/how-it-works',
+    text: 'How It Works'
```
Comment (stripedpajamas) [inline components/common/footer.js:20]: idk, i like it when websites have all their links in the footer. not sure of any reason to hide/show based on context

File: components/common/header/authNav.js (hunk 2 above)
Comment (stripedpajamas) [reply components/common/header/authNav.js]: i'd say leave the links in unless they are all in the Footer :D

File: components/splash/cards/whatIsFBCards.js (hunk 8 above)
Comment (stripedpajamas) [reply components/splash/cards/whatIsFBCards.js]: i don't like it either

Comment (meowwwls) [reply components/splash/cards/whatIsFBCards.js]: OK I'll revert that. Eryn noted it in her changes and it was in the original design, but I agree, I think it looks better how we had it.

File: components/common/header/authNav.js (hunk 2 above)
Comment (meowwwls) [reply components/common/header/authNav.js]: Reverted these changes.

File: components/dashboard/topTenPackagesView.js (hunk 4 above)
Comment (meowwwls) [reply components/dashboard/topTenPackagesView.js]: Done.

File: components/common/banner.js (hunk 7 above)
Comment (meowwwls) [reply components/common/banner.js:32]: I left it without the border for now since we're split 2-2 😬

File: components/common/footer.js (hunk 9 above)
Comment (meowwwls) [reply components/common/footer.js:20]: Changed it so the footer links are the same logged in and logged out (with the exception of "Log In" and "Sign Up")

File: components/common/header/authNav.js [hunk 10]
```diff
@@ -83,8 +83,7 @@ const AuthNav = ({ user, onLogout }) => {
             />
           </MenuButton>
           <MenuList backgroundColor='lightRock'>
-            {/* TODO: commenting these out for now in case we decide to add them back, but I agree with Eryn that they aren't needed */}
-            {/* <MenuItem
+            <MenuItem
```
Comment (joelwass) [inline components/common/header/authNav.js]: can we add the boxes to all of these like you did for settings and logout?

File: components/splash/cards/whatIsFBCards.js [hunk 11]
```diff
@@ -44,9 +44,10 @@ const Card = ({ icon, text, heading }) => (
       align='left'
       lineGap='1'
       lineColor='ocean'
-    >
-      <Icon name={icon} size='1.25rem' margin='.3rem .5rem .3rem 0' />
```
Comment (joelwass) [inline components/splash/cards/whatIsFBCards.js:48]: i think the icon needs to be within the underlineHeading

**Review by joelwass — CHANGES_REQUESTED** [review]

File: components/common/header/authNav.js (hunk 10 above)
Comment (meowwwls) [reply components/common/header/authNav.js]: Done

File: components/splash/cards/whatIsFBCards.js (hunk 11 above)
Comment (meowwwls) [reply components/splash/cards/whatIsFBCards.js:48]: Fixed!

**Review by meowwwls — COMMENTED** [review]
I adding handling for the "Contact Us" heading in the contact section. We need a heading because `sections` have to have a heading, and it also didn't look right not having it on the Contact Us page when it's on all other pages. 

So it's visually hidden now on the About Us page, but shows on the Contact Us page (with proper levels base on where it's used).

File: components/common/header/authNav.js [hunk 12]
```diff
@@ -114,8 +118,10 @@ const AuthNav = ({ user, onLogout }) => {
             >
               <Link href='/how-it-works'>
                 <a>
-                  <Icon name='info' marginRight='.5rem' />
-                  How It Works
+                  <Box as='span' display='flex' alignItems='center'>
+                    <Icon name='info' marginRight='.5rem' />
+                    <span>How It Works</span>
```
Comment (joelwass) [inline components/common/header/authNav.js]: i think we made these lower case after initial word

Comment (meowwwls) [reply components/common/header/authNav.js]: Done!

**Review by joelwass — APPROVED** [review]
